### PR TITLE
Programm becomes better and better

### PR DESCRIPTION
**Pull request by SubutaiBogatur**
I added:
1)Support of latin names of chemical elements. Now elements can be be called both by number of electrons and by name.
2)New text messages were added (2 messages).
3)Our programm started supporting elements with number of electrons from 1 to 49 only. As a result, documentation was created.

### CODE REVIEW

File: src/Atom.java [hunk 1]
```diff
@@ -7,6 +7,8 @@
         fillElectronicLayers(6, el);
     }
 
+
+
```
Comment (mutsch) [inline src/Atom.java]: Why did you add this strings?

File: src/Main.java [hunk 2]
```diff
@@ -38,11 +55,12 @@ static void update() {
                             els.append('f');
                             break;
                     }
-                    els.append("(" + a[i][j] + ")");
+                    els.append("(" + a[i][j] + ") ");
                 }
             }
         }
         elayers.setText(String.valueOf(els));
+        if(electro==0)elayers.setText("The element must have from 1 to 49 electrons.");
```
Comment (mutsch) [inline src/Main.java]: More spaces

**Comment by SubutaiBogatur** [comment]
accept immidiately
